code/modules/research/bepis.dm

```diff
@@ -61,6 +61,9 @@
 		return
 	if(istype(O, /obj/item/card/id))
 		var/obj/item/card/id/Card = O
+		if(istype(Card, /obj/item/card/id/departmental_budget)
```

Comment:
```suggestion
		if(istype(Card, /obj/item/card/id/departmental_budget))
```

Sir

Reply:
Oh I've had the fix as an uncommited change for about 7 hours now I'm just mulling about doing the rest of the PR better.